lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -138,7 +138,9 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
   /// \}
 
   /// A list of replaceable fwddecls that need to be RAUWed at the end.
-  std::vector<std::pair<TypeBase *, llvm::TrackingMDRef>> ReplaceMap;
+  std::vector<std::pair<TypeBase *, llvm::TrackingMDRef>> FwdDeclScopes;
```

Comment:
It doesn't look like anything is added in `FwdDeclScopes`?

Reply:
Thanks! That's a leftover from an earlier version of the patch.